pokemongo_bot/inventory.py

```diff
@@ -1134,9 +1139,16 @@ def update_web_inventory(self):
         if not os.path.exists(web_inventory):
             self.init_inventory_outfile()
             
-        with open(web_inventory, "r") as infile:
-            json_inventory = json.load(infile)
-            infile.close()
+        try:
```

Comment:
Latest commit excludes the infile.close() part (because it's useless) so basically it's just one line of code broken apart for ease of debugging. Not sure what the point of moving this would be.